modules/UI/recording/Recording.js

```diff
@@ -452,12 +450,13 @@ const Recording = {
     },
 
     // checks whether recording is enabled and whether we have params
-    // to start automatically recording
+    // to start automatically recording (XXX: No, it doesn't do that).
     checkAutoRecord() {
         if (_isRecordingButtonEnabled && config.autoRecord) {
             this.predefinedToken = UIUtil.escapeHtml(config.autoRecordToken);
-            this.eventEmitter.emit(UIEvents.RECORDING_TOGGLED,
-                                    this.predefinedToken);
+            this.eventEmitter.emit(
+                UIEvents.RECORDING_TOGGLED,
+                { token: this.predefinedToken });
```

Comment:
I'm assuming this was always broken then?

Reply:
:man_shrugging: